.pipelines/android_packaging.yml

```diff
@@ -19,57 +18,46 @@ jobs:
       - script: brew install coreutils ninja
         displayName: Install coreutils and ninja
 
-      - script: |
-          mkdir cmake/external
-          cd cmake/external
-          git clone https://github.com/microsoft/onnxruntime.git
-          cd onnxruntime
-          git checkout v1.12.1
-        displayName: Download Onnxruntime
-
-      - script: |
-          python tools/python/run_android_emulator.py \
-            --android-sdk-root ${ANDROID_SDK_ROOT} \
-            --create-avd --system-image "system-images;android-30;google_apis;x86_64" \
-            --start --emulator-extra-args="-partition-size 4096" \
-            --emulator-pid-file $(Build.BinariesDirectory)/emulator.pid
-        displayName: Start Android emulator
-        workingDirectory: $(OnnxruntimeDirectory)
-
-      - task: JavaToolInstaller@0
-        displayName: Use jdk 11
-        inputs:
-          versionSpec: "11"
-          jdkArchitectureOption: "x64"
-          jdkSourceOption: "PreInstalled"
-
       - script: |
           python ./tools/gen_selectedops.py ./tools/android/package_ops.config
```

Comment:
an alternative is to save a converted _selectedops.cmake in some folder and copy it to the /cmake folder, which can skip this conversions step.

Reply:
that would work too. I'll leave it like this now, would prefer to generate the file instead of checking in a generated file where possible.